compiler/internal/typeparams/instance.go

```diff
@@ -147,6 +147,18 @@ func (iset *InstanceSet) ByObj() map[types.Object][]Instance {
 	return result
 }
 
+// ForObj returns instances for a given object type belong to. Order is not specified.
+// This returns the same values as `ByObj()[obj]`.
```

Comment:
I'm no sure what this is meant to say. Not sure if it's missing a word, or has too many words.  :)  Please just re-read and edit for clarity.